src/adapt/vtree.js

```diff
@@ -672,7 +674,8 @@ adapt.vtree.newNodePositionFromNodeContext = function(nodeContext) {
         shadowType: adapt.vtree.ShadowType.NONE,
         shadowContext: nodeContext.shadowContext,
         nodeShadow: null,
-        shadowSibling: null
+        shadowSibling: null,
+        fragmentIndex: nodeContext.fragmentIndex
```

Comment:
fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/342/commits/f46ef89ab4302c9274b27ee3c06f734f5b67e252).